src/plugins/settings/bool-setting/bool-setting.less

```diff
@@ -1,14 +1,19 @@
 .uip-bool-setting {
   label {
     display: inline-flex;
+    flex-wrap: wrap;
     align-items: center;
 
     input {
-      margin: 5px 6px 2px;
+      margin: 5px 0 2px 6px;
+    }
+
+    span {
```

Comment:
Prefer CSS class to select this element instead of frequent tag selector that is potentially slow.